web/src/model/inventoryLogic.ts

```diff
@@ -0,0 +1,28 @@
+import Database from "../database";
+import Personnel from "./accounts/personnel";
+import Inventory from "./inventory";
+
+export default class InventoryLogic {
+    protected db: Database;
+
+    constructor(db: Database) {
+        this.db = db;
+    }
+
+    public async selectInventoryHardCode() {
```

Comment:
With respect to the rubric and work done, I can see why you chose this name, but I reckon someone who hasn't looked inside this file wouldn't know what this does. You could alternatively call it `selectInventory()` and parameterize it with booleans for each attribute.

The same goes for "inventory" vs. "inventoryLogic."

Not necessary for the project though. Best just to get it done. Thought I'd mention though. :)

Reply:
Just read your comment below, makes more sense to me I'll change some of the naming conventions! (: